style/imageButton.less

```diff
@@ -1,21 +1,20 @@
 .imageButton {
-	@titleColor: @accentColorDark;
-	@titleColorHover: @accentColorDim;
-	@borderColor: @grayBleached;
-
 	margin-bottom: 20px;
 	float: left;
 	width: 100%;
-	//max-width: 280px;
 	&.clear {
 		clear: left;
 	}
 	a {
-		color: @titleColor;
 		text-decoration: none;
+		h3 {
+			color: @accentColorDark;
+		}
```

Comment:
```suggestion
		color: @accentColorDark;
```

Reply:
It was color: @accentColorDark, but because h3 is set to black the accent color wasn't applying.